lib/rspec/rails/matchers/active_job.rb

```diff
@@ -169,6 +173,10 @@ def set_expected_number(relativity, count)
                                end
           end
 
+          def deserialize_arguments(job)
+            ::ActiveJob::Arguments.deserialize(job[:args]) rescue nil
```

Comment:
If we can't deserialize the args, the most we should do is return the original args.